commit 293d3393add7663b6c2f628b8618924c1f14df13 (refs/changes/21/821/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-31T12:24:53-07:00 (2 years, 11 months ago)
    
    DOCKER-960 include the docker-registry-client error in a NotFoundError docker pull error
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/lib/backends/sdc/images.js b/lib/backends/sdc/images.js
index b73b65f..835cdb2 100644
--- a/lib/backends/sdc/images.js
+++ b/lib/backends/sdc/images.js
@@ -1426,8 +1426,8 @@ function pullImage(opts, callback) {
             // E.g. `docker pull quay.io/no-such-user`.
             payload = {
                 status: format('Not Found (404) error from registry %s trying '
-                    + 'to pull %s (%s)', opts.rat.index.name,
-                    opts.rat.remoteName, opts.req_id)
+                    + 'to pull %s: %s (%s)', opts.rat.index.name,
+                    opts.rat.remoteName, err.message, opts.req_id)
             };
         } else if (err.code === 'DownloadError') {
             // E.g. A `DownloadError` from docker-registry-client.
diff --git a/package.json b/package.json
index 681eb91..ee3f6a3 100644
--- a/package.json
+++ b/package.json
@@ -1,13 +1,13 @@
 {
   "name": "sdc-docker",
-  "version": "0.3.1",
+  "version": "0.3.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
     "assert-plus": "1.0.0",
     "async": "0.9.0",
     "bunyan": "1.3.2",
-    "docker-registry-client": "3.2.3",
+    "docker-registry-client": "3.2.4",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "fwrule": "git+https://github.com/joyent/sdc-fwrule.git#d1174be",
     "imgmanifest": "git+https://github.com/joyent/node-imgmanifest.git#91e5d80",
